game/view/gameplay/actionhud/init.lua

```diff
@@ -371,11 +381,12 @@ function ActionHUD:update(dt)
   self.turnpreview:update(dt)
 
   -- Control hints
-  self.cancel_hint:update(dt)
-  self.confirm_hint:update(dt)
-  self.open_packs_hint:update(dt)
-  self.show_stats_hint:update(dt)
-  self.toggle_hints_hint:update(dt)
+  local control_hints = Util.findSubtype("control_hints")
+  if control_hints then
+    for button in pairs(control_hints) do
+      button:setMode(self.player_focused and "FOCUS" or "DEFAULT")
```

Comment:
more magical literals that would be really benefit to be stored somewhere else

Reply:
Alright, since these literals are directly tied to this PR, I'll move them to tables before merging.